rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -2259,6 +2259,11 @@ def test_upgrade_db_41_to_42(user_data_dir, messages_aggregator):
             ChainID.deserialize_from_name(x) for x in json.loads(raw_list)
         ] == [ChainID.POLYGON_POS, ChainID.GNOSIS]
 
+        # get settings and confirm manualcurrent is present before the upgrade
+        cursor.execute('SELECT value FROM settings WHERE name="current_price_oracles"')
+        oracles = json.loads(cursor.fetchone()[0])
```

Comment:
this test is failing here, did you try running it? It fails probably because this setting is not already present in the table of the DB that is being tested (v41_rotkehlchen.db).